documentation/modules/ROOT/pages/2-build-pipelines.adoc

```diff
@@ -114,14 +114,16 @@ For simplicity in this exercise, you will trigger the pipeline run manually.
 
 First, open a new browser to access the link:https://console-openshift-console.%SUBDOMAIN%/dev-pipelines/ns/cicd-%USERID%[OpenShift Pipeline^].
 
-Use the following credentials if you haven't logged in to the OpenShift cluster before.
+Use the following credentials if you haven't logged in to the OpenShift cluster before. OpenShift is already integrated with https://access.redhat.com/products/red-hat-single-sign-on/[Red Hat Single Sign On^].
 
-image::openshift_login.png[openshift_login]
+image::sso_login.png[openshift_login]
 
-Login using your credentials:
+* Login using your credentials:
 
-* Username: `%USERID%`
-* Password: `{openshift-password}`
+** Username: `%USERID%`
+** Password: `{openshift-password}`
+
+* Click on *Sign In*
 
```

Comment:
Will the login page redirect to the pipeline page automatically after login?

Reply:
yes

Reply:
Cool!